src/migrations/1653206298709-create_promocode_order_table.ts

```diff
@@ -0,0 +1,26 @@
+import type { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class createPromocodeOrderTable1653206298709
+    implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`
+        CREATE TABLE IF NOT EXISTS promocode_order
+        (
+            "id"            uuid              NOT NULL DEFAULT uuid_generate_v4(),
```

Comment:
we have one to one relations between order and promocode. how you plan to use this table? 

From this query seems that it will be more efficient to store promocode in order as seperate field

https://github.com/irnrogalik/nest_project/pull/6/files#diff-8ba8880b976e7c2393b403e8e591a01246d5b19081b490bfcb385d7fb2712bacR71

Also in order.entity you added promocode field, not reference to another entity https://github.com/irnrogalik/nest_project/pull/6/files#diff-b5a824ea919b96156d6bbac664313806fa8ac46ae723c74e7501099bdf5862a1R19